src/types/publication_types/artisticRegistration.types.ts

```diff
@@ -156,7 +156,7 @@ export interface Concert extends ArtisticOutputBase {
   extent: string;
   description: string;
   concertProgramme: MusicalWorkPerformance[];
-  partOfSeries: boolean;
+  concertSeries: string | undefined;
```

Comment:
Ser at backend definerer dette til `String`, så da kan vi fjerne `undefined` her også, som egentlig var ment som en workaround for vi flyttet checkboxen til state
https://github.com/BIBSYSDEV/nva-datamodel-java/blob/main/nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/music/Concert.java#L32
```suggestion
  concertSeries: string;
```